redash/destinations/pagerduty.py

```diff
@@ -0,0 +1,69 @@
+import pypd
+import logging
+
+from redash.destinations import *
+
+
+class Pagerduty(BaseDestination):
+
+    KEY_STRING = '{user_name}_{user_email}_{query_id}_{query_name}'
+    DESCRIPTION_STR = 'redash_{query_id}_{query_name}'
+
+    @classmethod
+    def configuration_schema(cls):
+        return {
+            'type': 'object',
+            'properties': {
+                'integration_key': {
+                    'type': 'string',
+                    'title': 'Pagerduty Service Integration Key'
+                },
+                'description': {
+                    'type': 'string',
+                    'title': 'Description for the event, defaults to query',
+                }
+            },
+            "required": ["integration_key"]
+        }
+
+    @classmethod
+    def icon(cls):
+        return 'creative-commons-pd-alt'
+
+    def notify(self, alert, query, user, new_state, app, host, options):
+
+        default_desc = self.DESCRIPTION_STR.format(query_id=query.id, query_name=query.lowercase_name)
```

Comment:
Following the comment about the description text, I don't think we need to lower case the query name.